lib/src/api/magazine_moderation.dart

```diff
@@ -22,7 +22,16 @@ class APIMagazineModeration {
         throw Exception('List banned users not allowed on lemmy');
 
       case ServerSoftware.piefed:
-        throw UnimplementedError();
+        final path = '/community/moderate/bans';
+
+        final body = {
+          'community_id': magazineId,
+          'page': (page == null) ? 1 : page,
```

Comment:
The `??` operator checks if _left_ is not `null` and uses it if that's true, otherwise, if it is `null`, it falls back to _right_.

```suggestion
          'page': page ?? 1,
```

Reply:
Oh, I wasn't saying it shouldn't be paged. I think it should.

I was just asking if the `page` property was optional in the request body (where it would default to `1` if not provided)?

Reply:
yeah it does default to `1` if `page` is not in the request, but i kept getting errors when sending the initial call from interstellar when `page` was null, which translated to `None` on the server side.

the firstPageKey starts as `''` here: https://github.com/JollyDevelopment/interstellar/blob/jollydev/piefed-community-moderation-list-bans-unban/lib/src/screens/explore/magazine_mod_panel.dart#L78 

so I figured I could check if its the first call and if so /not/ send the `page` in the body, or just send `page:1` if its the first call. 

one of those half-dozen-of-one six-of-another sort of calls.